### PR TITLE
Run the code formattter on lib/mix/lib/mix/generator.ex

### PR DESCRIPTION
**Pull request by sger**

### CODE REVIEW

File: lib/mix/lib/mix/generator.ex [hunk 1]
```diff
@@ -77,15 +77,20 @@ defmodule Mix.Generator do
...
           _ ->
             raise ArgumentError, "expected string or from_file: file"
         end
 
       require EEx
-      EEx.function_from_string :defp, :"#{name}_template", "<% _ = assigns %>" <> contents, [:assigns]
+
+      EEx.function_from_string(:defp, :"#{name}_template", "<% _ = assigns %>" <> contents, [
+        :assigns
+      ])
```
Comment (ericmj) [inline lib/mix/lib/mix/generator.ex]: Please break `"<% _ = assigns %>" <> contents` out into a variable so the function call fits on one line.

Comment (sger) [reply lib/mix/lib/mix/generator.ex]: Please check it @ericmj first time touching elixir code so i need your guidance.
Thanks!

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Comment by sger** [comment]
thanks @josevalim

**Pull request merged**